.travis.yml

```diff
@@ -22,7 +22,7 @@ install:
   - conda info -a
 
   # Replace dep1 dep2 ... with your dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION scipy numpy
```

Comment:
Actually, scikit-learn is explicitly designed in a way that you don't need to depend on it to satisfy it's API (you need to implement ``set_params`` and ``get_params``, though that is very easy).